stan-dev/cmdstan
# Fix test deprecation warnings

#### Submisison Checklist


#### Summary:

Fixes deprecation warnings in test models.

#### How to Verify:

Before this PR, if you run

~~~
make clean
./runCmdStanTests.py src/test 
~~~

you should see

~~~
Model name=csv_header_consistency_model                                                                                      
Input file=src/test/test-models/csv_header_consistency.stan                                                                  
Output file=src/test/test-models/csv_header_consistency.hpp
DIAGNOSTIC(S) FROM PARSER:                                     
Warning (non-fatal): assignment operator <- deprecated in the Stan language; use = instead.
Warning (non-fatal): assignment operator <- deprecated in the Stan language; use = instead.
Warning (non-fatal): assignment operator <- deprecated in the Stan language; use = instead.                                   
Warning (non-fatal): assignment operator <- deprecated in the Stan language; use = instead.                                   
Warning (non-fatal): assignment operator <- deprecated in the Stan language; use = instead.                                   
Warning (non-fatal): assignment operator <- deprecated in the Stan language; use = instead.                                   
Warning (non-fatal): assignment operator <- deprecated in the Stan language; use = instead.                                   
Warning (non-fatal): assignment operator <- deprecated in the Stan language; use = instead.                              

...

Model name=value_fail_model                                    
Input file=src/test/test-models/value_fail.stan                                                                               
Output file=src/test/test-models/value_fail.hpp                
DIAGNOSTIC(S) FROM PARSER:                                     
Warning (non-fatal): increment_log_prob(...); is deprecated and will be removed in the future.
~~~

With this PR, you shouldn't see those warnings.

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company):

Dan Luu

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
